content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -847,6 +847,35 @@ not vulnerable to ordering changes in the list.
 
 Once the last finalizer is removed, the resource is actually removed from etcd.
 
+### Force deletion
+
+{{< feature-state feature_gate_name="AllowUnsafeMalformedObjectDeletion" >}}
+
+{{< caution >}}
+This may break the workload associated with the resource being force deleted, if it
+relies on the normal deletion flow, so cluster breaking consequences may apply.
+{{< /caution >}}
+
+By enabling the delete option `ignoreStoreReadErrorWithClusterBreakingPotential`, the
+user can perform an unsafe, and force **delete** operation of an undecryptable/corrupt
+resource. This option is behind a new ALPHA feature gate, and it is disabled by
+default. In order to use this option, the cluster operator must enable the feature by
+setting the command line option `--feature-gates=AllowUnsafeMalformedObjectDeletion=true`.
+
+{{< note >}}
+The user performing the force **delete** operation must have the
+`unsafe-delete-ignore-read-errors` permission on the resource.
+{{< /note >}}
+
+A resource is considered corrupt if it can not be successfully retrieved from the
+storage due to a) transformation error e.g. decryption failure, or b) the object
```

Comment:
We [avoid Latin phrases and their abbreviations](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-latin-phrases)
```suggestion
storage due to a) transformation error (for example: decryption failure), or b) the object
```